app/src/main/res/values/strings.xml

```diff
@@ -20,14 +20,13 @@
     <string name="mode_animation">Animation</string>
     <string name="mode_laser">Laser</string>
     <string name="send_button">SEND</string>
-    <string name="preview_button">PREVIEW</string>
     <string name="permission_required">Permission required</string>
-    <string name="txt_turn_on_bluetooth">Turn on Bluetooth and try again</string>
```

Comment:
Same

Reply:
Yes Sir, sure. let me undo that

Reply:
Is this resource really needed @naman653? We are using `enable_bluetooth` to warn the user when bluetooth is off. This string is redundant in any case according to me.

Reply:
@naman653 Yeah TBH @yashk2000 is right